src/push.go

```diff
@@ -209,15 +204,34 @@ func getOrCreateGitHubRepo(ctx context.Context, client *github.Client, repoName,
 		}
 	}
 
-	ghRepo, resp, err := client.Repositories.Create(ctx, createRepoOrgName, repo)
+	// check if repository already exists
+	ghRepo, resp, err := client.Repositories.Get(ctx, ownerName, repoName)
```

Comment:
can we do err != nil check also and trace the error

Reply:
I felt I missed some branch :D. What I found out while working on the E2E tests, the go-github client is hiding quite a lot of error information from the API result :/ But that is a general issue.